meetings/osf_oauth2_adapter/migrations/0001_add_site_and_social_app.py

```diff
@@ -7,11 +7,11 @@
 from allauth.socialaccount.models import SocialApp
 from django.contrib.sites.models import Site
 
-from osf_oauth2_adapter.apps import OsfOauth2AdapterConfig
-from django.conf import settings
+from meetings.utils import OsfOauth2AdapterConfig
```

Comment:
This seems like a very strange import for the `osf_oauth2_adapter` to be in `meetings.utils` when it was in its own directory prior to this change.